way_to_home/tests/unittests/custom_user/testmodel.py

```diff
@@ -0,0 +1,116 @@
+"""
+CustomUser Model Test
+========================
+This module provides complete testing for all CustomUser's model functions.
+"""
+
+from django.test import TestCase
+from custom_user.models import CustomUser
+
+
+class CustomUserTestCase(TestCase):
```

Comment:
```suggestion
class CustomUserTestCase(TestCase):
```
It is better to change to `CustomUserModelTestCase`.

Reply:
fixed